Listing '.'...
Listing './bin'...
Listing './include'...
Listing './lib'...
Listing './lib/python3.5'...
Listing './lib/python3.5/site-packages'...
Listing './lib/python3.5/site-packages/archgenxml'...
Compiling './lib/python3.5/site-packages/archgenxml/ArchetypesGenerator.py'...
***   File "./lib/python3.5/site-packages/archgenxml/ArchetypesGenerator.py", line 815
    except Exception, e:
                    ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/archgenxml/BaseGenerator.py'...
***   File "./lib/python3.5/site-packages/archgenxml/BaseGenerator.py", line 196
    raise ValueError, message
                    ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/archgenxml/OptionParser.py'...
***   File "./lib/python3.5/site-packages/archgenxml/OptionParser.py", line 98
    print parser.sample_config()
               ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/archgenxml/PyParser.py'...
***   File "./lib/python3.5/site-packages/archgenxml/PyParser.py", line 205
    print 'PyModule:'
                    ^
SyntaxError: Missing parentheses in call to 'print'

Compiling './lib/python3.5/site-packages/archgenxml/TaggedValueSupport.py'...
***   File "./lib/python3.5/site-packages/archgenxml/TaggedValueSupport.py", line 1351
    print tgvRegistry.documentation()
                    ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/archgenxml/UMLProfile.py'...
***   File "./lib/python3.5/site-packages/archgenxml/UMLProfile.py", line 51
    raise KeyError,key
                  ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/archgenxml/patches.py'...
***   File "./lib/python3.5/site-packages/archgenxml/patches.py", line 33
    except KeyError, v:
                   ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/archgenxml/plone'...
Listing './lib/python3.5/site-packages/archgenxml/templates'...
Listing './lib/python3.5/site-packages/archgenxml/templates/archetypes'...
Listing './lib/python3.5/site-packages/archgenxml/templates/profiles'...
Listing './lib/python3.5/site-packages/archgenxml/templates/tests'...
Listing './lib/python3.5/site-packages/archgenxml/tests'...
Listing './lib/python3.5/site-packages/archgenxml/tests/models'...
Listing './lib/python3.5/site-packages/archgenxml/tests/pythonfiles'...
Compiling './lib/python3.5/site-packages/archgenxml/utils.py'...
***   File "./lib/python3.5/site-packages/archgenxml/utils.py", line 216
    print '***'
              ^
SyntaxError: Missing parentheses in call to 'print'

Compiling './lib/python3.5/site-packages/archgenxml/uuidfrompy252.py'...
***   File "./lib/python3.5/site-packages/archgenxml/uuidfrompy252.py", line 151
    if not 0 <= time_low < 1<<32L:
                                ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/archgenxml-2.7.egg-info'...
Listing './lib/python3.5/site-packages/i18ndude'...
Compiling './lib/python3.5/site-packages/i18ndude/extract.py'...
***   File "./lib/python3.5/site-packages/i18ndude/extract.py", line 528
    print 'There was an error processing', filename
                                        ^
SyntaxError: Missing parentheses in call to 'print'

Compiling './lib/python3.5/site-packages/i18ndude/gsextract.py'...
***   File "./lib/python3.5/site-packages/i18ndude/gsextract.py", line 26
    print u"There was an error in parsing %s: %s" % (filename, e)
                                                ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/i18ndude/pygettext.py'...
***   File "./lib/python3.5/site-packages/i18ndude/pygettext.py", line 578
    print _('pygettext.py (xgettext for Python) %s') % __version__
          ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/i18ndude/script.py'...
***   File "./lib/python3.5/site-packages/i18ndude/script.py", line 456
    print '%s: %s added, %s removed' % (po.filename,
                                   ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/i18ndude/tests'...
Listing './lib/python3.5/site-packages/i18ndude/tests/input'...
Listing './lib/python3.5/site-packages/i18ndude/tests/output'...
Compiling './lib/python3.5/site-packages/i18ndude/utils.py'...
***   File "./lib/python3.5/site-packages/i18ndude/utils.py", line 245
    print "Wrote command line documentation to", target
                                              ^
SyntaxError: Missing parentheses in call to 'print'

Compiling './lib/python3.5/site-packages/i18ndude/visualisation.py'...
***   File "./lib/python3.5/site-packages/i18ndude/visualisation.py", line 108
    print template
                 ^
SyntaxError: Missing parentheses in call to 'print'

Compiling './lib/python3.5/site-packages/i18ndude/zcmlextract.py'...
***   File "./lib/python3.5/site-packages/i18ndude/zcmlextract.py", line 55
    print u"There was an error in parsing %s: %s" % (filename, e)
                                                ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/i18ndude-4.3.egg-info'...
Listing './lib/python3.5/site-packages/lxml'...
Listing './lib/python3.5/site-packages/lxml/html'...
Listing './lib/python3.5/site-packages/lxml/includes'...
Listing './lib/python3.5/site-packages/lxml/includes/libexslt'...
Listing './lib/python3.5/site-packages/lxml/includes/libxml'...
Listing './lib/python3.5/site-packages/lxml/includes/libxslt'...
Listing './lib/python3.5/site-packages/lxml/isoschematron'...
Listing './lib/python3.5/site-packages/lxml/isoschematron/resources'...
Listing './lib/python3.5/site-packages/lxml/isoschematron/resources/rng'...
Listing './lib/python3.5/site-packages/lxml/isoschematron/resources/xsl'...
Listing './lib/python3.5/site-packages/lxml/isoschematron/resources/xsl/iso-schematron-xslt1'...
Listing './lib/python3.5/site-packages/lxml-4.1.1.dist-info'...
Listing './lib/python3.5/site-packages/ordereddict-1.1.egg-info'...
Listing './lib/python3.5/site-packages/pip'...
Listing './lib/python3.5/site-packages/pip/_vendor'...
Listing './lib/python3.5/site-packages/pip/commands'...
Listing './lib/python3.5/site-packages/pip/compat'...
Listing './lib/python3.5/site-packages/pip/models'...
Listing './lib/python3.5/site-packages/pip/operations'...
Listing './lib/python3.5/site-packages/pip/req'...
Listing './lib/python3.5/site-packages/pip/utils'...
Listing './lib/python3.5/site-packages/pip/vcs'...
Listing './lib/python3.5/site-packages/pip-9.0.1.dist-info'...
Listing './lib/python3.5/site-packages/pkg_resources'...
Listing './lib/python3.5/site-packages/pkg_resources/_vendor'...
Listing './lib/python3.5/site-packages/pkg_resources/_vendor/packaging'...
Listing './lib/python3.5/site-packages/pkg_resources/extern'...
Listing './lib/python3.5/site-packages/pkg_resources-0.0.0.dist-info'...
Listing './lib/python3.5/site-packages/setuptools'...
Listing './lib/python3.5/site-packages/setuptools/command'...
Listing './lib/python3.5/site-packages/setuptools/extern'...
Listing './lib/python3.5/site-packages/setuptools-32.3.1.dist-info'...
Listing './lib/python3.5/site-packages/stripogram'...
Compiling './lib/python3.5/site-packages/stripogram/HTMLParser.py'...
***   File "./lib/python3.5/site-packages/stripogram/HTMLParser.py", line 198
    "invalid '<' construct: %s" % `rawdata[i:i+2]`,
                                  ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/stripogram/tests.py'...
*** Sorry: TabError: inconsistent use of tabs and spaces in indentation (tests.py, line 34)
Listing './lib/python3.5/site-packages/stripogram-1.5.egg-info'...
Listing './lib/python3.5/site-packages/xmiparser'...
Listing './lib/python3.5/site-packages/xmiparser/doc'...
Listing './lib/python3.5/site-packages/xmiparser/tests'...
Compiling './lib/python3.5/site-packages/xmiparser/xmiparser.py'...
***   File "./lib/python3.5/site-packages/xmiparser/xmiparser.py", line 400
    raise TypeError, 'element %s has empty taggedValue' % self.getId(el)
                   ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/xmiparser-1.5.egg-info'...
Listing './lib/python3.5/site-packages/zope'...
Listing './lib/python3.5/site-packages/zope/component'...
Listing './lib/python3.5/site-packages/zope/component/testfiles'...
Listing './lib/python3.5/site-packages/zope/component/tests'...
Listing './lib/python3.5/site-packages/zope/documenttemplate'...
Compiling './lib/python3.5/site-packages/zope/documenttemplate/dt_let.py'...
***   File "./lib/python3.5/site-packages/zope/documenttemplate/dt_let.py", line 64
    except SyntaxError, v:
                      ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/zope/documenttemplate/dt_string.py'...
***   File "./lib/python3.5/site-packages/zope/documenttemplate/dt_string.py", line 99
    exec 'from %s import %s' % (module, name) in d
                           ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/zope/documenttemplate/dt_util.py'...
***   File "./lib/python3.5/site-packages/zope/documenttemplate/dt_util.py", line 107
    except SyntaxError, v:
                      ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/zope/documenttemplate/dt_var.py'...
***   File "./lib/python3.5/site-packages/zope/documenttemplate/dt_var.py", line 264
    raise 'Document Error',(
                          ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/zope/documenttemplate/pdocumenttemplate.py'...
***   File "./lib/python3.5/site-packages/zope/documenttemplate/pdocumenttemplate.py", line 299
    except KeyError, v:
                   ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/zope/documenttemplate/tests'...
Compiling './lib/python3.5/site-packages/zope/documenttemplate/tests/dtmltestbase.py'...
***   File "./lib/python3.5/site-packages/zope/documenttemplate/tests/dtmltestbase.py", line 39
    return "D(%s)" % `self.__dict__`
                     ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/zope/documenttemplate/tests/testustr.py'...
***   File "./lib/python3.5/site-packages/zope/documenttemplate/tests/testustr.py", line 43
    assert a=='hello', `a`
                       ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/zope/documenttemplate/untrusted'...
Listing './lib/python3.5/site-packages/zope/event'...
Listing './lib/python3.5/site-packages/zope/i18nmessageid'...
Listing './lib/python3.5/site-packages/zope/interface'...
Listing './lib/python3.5/site-packages/zope/interface/common'...
Listing './lib/python3.5/site-packages/zope/interface/common/tests'...
Listing './lib/python3.5/site-packages/zope/interface/tests'...
Listing './lib/python3.5/site-packages/zope/structuredtext'...
Listing './lib/python3.5/site-packages/zope/structuredtext/regressions'...
Listing './lib/python3.5/site-packages/zope/tal'...
Listing './lib/python3.5/site-packages/zope/tal/benchmark'...
Listing './lib/python3.5/site-packages/zope/tal/tests'...
Listing './lib/python3.5/site-packages/zope/tal/tests/input'...
Listing './lib/python3.5/site-packages/zope/tal/tests/output'...
Listing './lib/python3.5/site-packages/zope.component-4.4.1.dist-info'...
Listing './lib/python3.5/site-packages/zope.documenttemplate-3.4.3.egg-info'...
Listing './lib/python3.5/site-packages/zope.event-4.3.0.dist-info'...
Listing './lib/python3.5/site-packages/zope.i18nmessageid-4.1.0.dist-info'...
Listing './lib/python3.5/site-packages/zope.interface-4.4.3.dist-info'...
Listing './lib/python3.5/site-packages/zope.structuredtext-4.2.0.dist-info'...
Listing './lib/python3.5/site-packages/zope.tal-4.3.0.dist-info'...
Listing './share'...
Listing './share/python-wheels'...
